### ISSUE TITLE
Sample data deploy fails on 4.0.4 version with Magento 2.3.4

### ISSUE THREAD

**Issue by easysoft-team** [netz98/n98-magerun2#541]
```
n98-magerun2.phar sampledata:deploy
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)

Installation failed, reverting ./composer.json to its original content.

                                                                                           
  [ErrorException]                                                                         
  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?  
                                                                                           

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] [<packages>]...

There is an error during sample data deployment. Composer file will be reverted.
```

With `bin/magento sampledata:deploy` works fine

**Comment by cmuench** [comment]
@easysoft-team 
That's a well known problem of the Magento Sample Data.
There is a open issue for that. If you run the sample data import with PHP 7.3 it's broken.
Could it be that you run bin/magento with PHP 7.2 and n98-magerun2 with PHP 7.3?

**Comment by easysoft-team** [comment]
Hi @cmuench ,

I am using `PHP 7.2.28` 

megerun2 throws Exception, bin/magento is working

**Comment by cmuench** [comment]
@easysoft-team can you run n98-magerun2 with `-vvv` option?
This prints the stack trace of the exception.
Would be interesting to see where the exception is throwed.

**Comment by easysoft-team** [comment]
@cmuench ,
ok version was 4.0.3 and maybe PHP 7.3.
Right now I am using PHP 7.2 and version 4.0.4 got this:

`
magerun2 --version
n98-magerun2 4.0.4 by netz98 GmbH
user@magento2-web:/var/www/html$ magerun2 sampledata:deploy
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
PHP Fatal error:  Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21

Fatal error: Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21
`

**Comment by easysoft-team** [comment]
on 4.0.3 same issue:

```
magerun2 --version
n98-magerun2 4.0.3 by netz98 GmbH
user@magento2-web:/var/www/html$ magerun2 sampledata:deploy
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
PHP Fatal error:  Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21

Fatal error: Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21
```

**Comment by easysoft-team** [comment]
sampledata:reset is working, but same issue with sampledata:remove

```
 magerun2 sampledata:reset
Reset of sample data version completed successfully.
user@magento2-web:/var/www/html$ magerun2 sampledata:remove
Loading composer repositories with package information
Updating dependencies (including require-dev)
PHP Fatal error:  Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21

Fatal error: Class Composer\Package\AliasPackage contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Composer\Package\CompletePackageInterface::getFunding) in phar:///usr/local/bin/magerun2/vendor/composer/composer/src/Composer/Package/AliasPackage.php on line 21
```

**Comment by cmuench** [comment]
@easysoft-team thanks for the information. Will check that tomorrow.

**Comment by cmuench** [comment]
@easysoft-team I was not able to reproduce the problem on my machine. Worked also with PHP 7.3. Nevertheless I was able to figure out that you should have a Composer version >1.9 in your Magento installation. Magerun comes with 1.9.3 which was the up to date version if Magento 2.3.4 was released.
I updated the Composer version in n98-magerun2. It's deployed as new development version.
You can try it by updating with unstable option.

In your case:

`magerun2 self-update --unstable`

**Comment by easysoft-team** [comment]
Hi @cmuench 
I am using PHP 7.2.28
Composer is 1.9.3
after updating Magerun to 4.1.0 version, issue is gone and working perfect.
Waiting for stable release 4.1.0.
Thank you.

**Comment by cmuench** [comment]
Released in v4.1.0.
